docs/quickstart-aws.md

```diff
@@ -179,24 +179,34 @@ kubectl --kubeconfig=<cluster_name>-kubeconfig
 ```
 
 or export the `KUBECONFIG` variable environment variable:
+
 ```bash
 export KUBECONFIG=$PWD/<cluster_name>-kubeconfig
 ```
 
 ## Scaling Worker Nodes
 
-As worker nodes are managed by machine-controller, they can be scaled up and down
-(including to 0) using Kubernetes API.
+As worker nodes are managed by machine-controller. It creates initially one per
```

Comment:
There is something missing in the first sentence.

Reply:
Oh, yeah, missed it during reformulation. 😅